time stamp
float64[] x
float64[] th
float64[] x_input

float64 vel_out
float64 theta_out